### PR TITLE
Birds: faces & social links

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- /about/
- /authors/jonny/
- /authors/miriam/

## Show me
_Provide screenshots/animated gifs/videos if necessary._

about page with all the oddbird faces, and then jonny & mia bios

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Jonny wants an updated sketch. 🙏](https://trello.com/c/h3ObWNTr/161-jonny-wants-an-updated-sketch-%F0%9F%99%8F)

**Comment by mirisuzanne** [comment]
[I want rel=me on bird social links](https://trello.com/c/7oyJ3Uvg/202-i-want-relme-on-bird-social-links)

File: content/_includes/contact.macros.njk [hunk 1]
```diff
@@ -144,6 +144,20 @@ params:
...
+        {%- set text = text + context if not details -%}
+        <li>
+          {{- utility.icon_link(
+            icon='social/mastodon',
+            text=text | safe,
+            url=['https://', id.server, '/@', id.user] | join,
+            class='u-url',
+            attrs={'rel': rel} if rel else {}
+          ) -}}
+          {{- context if details -}}
```
Comment (jgerigmeyer) [inline content/_includes/contact.macros.njk:159]: I notice we never use `details=true` with this macro, so we could probably remove it and simplify this.

File: content/birds/miriam.md [hunk 2]
```diff
@@ -12,10 +12,11 @@ image:
     and white-blue-pink lightning-bolt earrings
   position: top
 social:
-  twitter: TerribleMia
```
Comment (jgerigmeyer) [inline content/birds/miriam.md:15]: This replaces `@TerribleMia` with `@oddbird` in the `<meta name="twitter:creator">` tag on all your posts, but that's probably fine?

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
Jonny's face LGTM!

I also have a Mastodon account and don't use Dribbble, so Mastodon could replace my Dribbble account on the About page. https://mastodon.social/@sondrae
I wonder whether anyone else's social media links need to be updated while we're at it.

Do we want to replace the OddBird Twitter and LinkedIn team image too (linkedin is really out of date) or do we want to switch social media platforms for OddBird? (We also still have an OddBird Facebook page.)

File: content/birds/miriam.md (hunk 2 above)
Comment (mirisuzanne) [reply content/birds/miriam.md:15]: Yeah, I suppose that's ok.

**Pull request merged**